common/to.etc.alg/pom.xml

```diff
@@ -51,7 +51,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
We do not need this.

Reply:
? It has handy methods that I use to encode to hex and to decode from hex. Why we should not use it?